maps/submaps/surface_submaps/wilderness/Blackshuttledown.dmm

```diff
@@ -5,11 +5,11 @@
 "ae" = (/obj/effect/decal/cleanable/blood,/turf/template_noop,/area/submap/Blackshuttledown)
```

Comment:
idk how they are doing wilderness but it's probably better than my old system

Reply:
Their southern cross map, I'm not 100% sure how it works. I played on a local server briefly and it has a station in orbit and a shuttle you can take to the surface, land at outposts and go from there. Kinda wanna do a similar thing for us but since we have the tether, instead there's like... a ground-effect hovercraft sort of low plane thing that you can take to wilderness?

Reply:
That would be nice.